assets/javascripts/running.js

```diff
@@ -262,6 +262,21 @@ function initLivelog() {
 
 /********* LIVE LOG END *********/
 
+/********* LIVE TERMINAL *********/
```

Comment:
I was just copying the existing code, I don't like the comments, but I assumed it was the preferred style of whoever maintains the JS the most.